recipe/meta.yaml

```
@@ -1,108 +1,113 @@
+    - scipy >=1.13.0,<1.16.0  # [osx and x86_64]
     - threadpoolctl >=3.1.0
-    - llvm-openmp         # [osx]
-    - _openmp_mutex       # [linux]
+    - llvm-openmp  # [osx]
+    - _openmp_mutex  # [linux]
+  run_constrained:
     # mkl variant pulls in intel-openmp which conflicts with
     # llvm-openmp. i.e. force to use openblas variant of numpy, scipy,
     # etc.
-    - blas * openblas     # [osx and x86_64]
-
-{% set tests_to_skip = "_not_a_real_test" %}
-
-# The following test seems to be crashing CI OS.
-{% set tests_to_skip = tests_to_skip + " or test_non_negative_factorization_consistency" %}         # [osx and arm64]
-
-# ValueError: Buffer dtype mismatch, expected 'intp_t' but got 'long'
-{% set tests_to_skip = tests_to_skip + " or test_sort_log2_build" %}                                # [win]
-
-# Only happened on py39, might be a timing issue as it fails at "Check if score and timing are reasonable".
-{% set tests_to_skip = tests_to_skip + " or test_random_search_cv_results_multimetric" %}           # [linux and aarch64]
+    - blas=*=openblas  # [osx and x86_64]
 
 test:
-  requires:
-    - pip
-    - pytest >=7.1.2
-    - pytest-timeout
-    - pytest-xdist
-    - matplotlib-base >=3.3.4
-    - scikit-image >=0.17.2
-    - pandas >=1.1.5
-    # No polars
-    #- polars >=0.20.23
-    - pyarrow >=12.0.0
-    # recent numpydoc not yet available
-    - numpydoc >=1.2.0  # [py<313]
-    - pooch >=1.6.0
   imports:
     - sklearn
     - sklearn.cluster
+    - sklearn.compose
+    - sklearn.covariance
+    - sklearn.cross_decomposition
     - sklearn.datasets
     - sklearn.decomposition
     - sklearn.ensemble
     - sklearn.feature_extraction
+    - sklearn.feature_selection
+    - sklearn.frozen
+    - sklearn.gaussian_process
+    - sklearn.impute
+    - sklearn.inspection
     - sklearn.linear_model
     - sklearn.manifold
     - sklearn.metrics
+    - sklearn.mixture
+    - sklearn.model_selection
     - sklearn.neighbors
+    - sklearn.neural_network
+    - sklearn.preprocessing
+    - sklearn.semi_supervised
+    - sklearn.svm
     - sklearn.tree
     - sklearn.utils
-    - sklearn.svm
   commands:
-    - pip check
-    - set OPENBLAS_NUM_THREADS=1          # [win]
-    - set OMP_NUM_THREADS=1               # [win]
-    - export OPENBLAS_NUM_THREADS=1       # [not win]
-    - export OMP_NUM_THREADS=1            # [not win]
-    - pytest --timeout 1200 -nauto --verbose --pyargs sklearn -k "not ({{ tests_to_skip }})"
+    #- pip check
+    - set OPENBLAS_NUM_THREADS=1  # [win]
+    - set OMP_NUM_THREADS=1  # [win]
+    - export OPENBLAS_NUM_THREADS=1  # [not win]
+    - export OMP_NUM_THREADS=1  # [not win]
+    # test_docstring
+    # >   return ".".join([self.obj.__module__, self.obj.__name__])
+    # E   AttributeError: 'property' object has no attribute '__module__'. Did you mean: '__reduce__'?
+    #- pytest --timeout 1200 -nauto --verbose --pyargs sklearn -k "not (test_docstring)"
```

```suggestion
    # test_docstring
    # >   return ".".join([self.obj.__module__, self.obj.__name__])
    # E   AttributeError: 'property' object has no attribute '__module__'. Did you mean: '__reduce__'?
    - pytest --timeout 1200 -nauto --verbose --pyargs sklearn -k "not (test_docstring)"
```
Please re-enable tests.

The comment does not give information as to where the error was. Was it in numpydoc?...